spynnaker/pyNN/models/neuron/synapse_dynamics/synapse_dynamics_structural_common.py

```diff
@@ -14,55 +14,23 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import collections
-import numpy as np
-from six import itervalues
+import numpy
 from data_specification.enums.data_type import DataType
 from spynnaker.pyNN.models.neural_projections import ProjectionApplicationEdge
-from spynnaker.pyNN.models.neural_projections import ProjectionMachineEdge
 from .abstract_synapse_dynamics_structural import (
     AbstractSynapseDynamicsStructural)
+from spynnaker.pyNN.exceptions import SynapticConfigurationException
 from spynnaker.pyNN.utilities import constants
+import math
 
 
-class SynapseDynamicsStructuralCommon(AbstractSynapseDynamicsStructural):
+class SynapseDynamicsStructuralCommon(object):
     """ Common class that enables synaptic rewiring. It acts as a wrapper\
         around SynapseDynamicsStatic or SynapseDynamicsSTDP.\
         This means rewiring can operate in parallel with these\
         types of synapses.
 
         Written by Petrut Bogdan.
```

Comment:
I think this is a legacy comment and should be updated. SPDynamisCommon no longer wraps around SPDynamicsSTDP or SPDyanmicsStatic. It now is more of a utility class which generates the appropriate data to load onto SpiNNaker.